code/modules/recycling/disposal.dm

```diff
@@ -663,8 +662,6 @@
 			if(M.client)	// if a client mob, update eye to follow this holder
 				M.client.eye = src
 
-	if(other.has_fat_guy)
```

Comment:
wouldn't this make it so that you can't have a bunch of people get stuck a-la futurama pneumo tubes?

Bit nonatomic

Reply:
The merged holder will inherit the fat mobs and I'm making it check for them during the movement itself now so it shouldn't change that.